doc/architecture.rst

```diff
@@ -179,21 +179,27 @@ information recording the overall health of the Ceph Storage Cluster.
 High Availability Monitors
 ~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Before Ceph Clients can read or write data, they must contact a Ceph Monitor
-to obtain the most recent copy of the cluster map. A Ceph Storage Cluster
-can operate with a single monitor; however, this introduces a single 
-point of failure (i.e., if the monitor goes down, Ceph Clients cannot
-read or write data).
-
-For added reliability and fault tolerance, Ceph supports a cluster of monitors.
-In a cluster of monitors, latency and other faults can cause one or more
-monitors to fall behind the current state of the cluster. For this reason, Ceph
-must have agreement among various monitor instances regarding the state of the
-cluster. Ceph always uses a majority of monitors (e.g., 1, 2:3, 3:5, 4:6, etc.)
-and the `Paxos`_ algorithm to establish a consensus among the monitors about the
-current state of the cluster.
-
-For details on configuring monitors, see the `Monitor Config Reference`_.
+A Ceph Client must contact a Ceph Monitor and obtain a current copy of the
+cluster map in order to be permitted to read data from or to write data to the
+Ceph cluster.
+
+It is possible for a Ceph cluster to function properly with only a single
+monitor in it, but a Ceph cluster that has only a single monitor in it has a
+single point of failure: if the monitor goes down, Ceph clients will be unable
+to read data from or write data to the cluster.
+
+Ceph supports a cluster of monitors in order to increase reliability and fault
```

Comment:
s/supports/leverages/

Reply:
This is that rarest of cases: the time when I think that "leveraged" is, in fact, the right word.

Accepted.

Reply:
Ever since I read that "utilize" doesn't mean what I think it means I've shied away from it.